tests/transactional/enable_selinux.pm

```diff
@@ -13,11 +13,28 @@ use strict;
 use warnings;
 use testapi;
 use transactional qw(process_reboot);
-use version_utils qw(is_sle_micro is_public_cloud);
+use version_utils qw(is_leap_micro is_sle_micro is_public_cloud);
 use Utils::Systemd qw(systemctl);
 use Utils::Logging 'save_and_upload_log';
 use transactional qw(trup_call check_reboot_changes process_reboot);
 
+sub check_enabled {
```

Comment:
```suggestion
sub check_enforcing {
```

This routine checks not only if SELinux is enabled, but if it is enforcing. I would rename the routine accordingly.